docs/ossec_alerts.rst

```diff
@@ -550,10 +599,18 @@ like the following: ::
       The iptables default drop rules are incorrect.
 
 Alternatively, the error text may say: ``The iptables rules have not been configured.``
-
 To resolve the issue, you can reinstate the standard iptables
 rules by :ref:`updating the system configuration <update-system-configuration>`.
 
+``securedrop-admin`` commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+OSSEC will send an alert when there is change to the system configuration or
+backups, using the command-line `securedrop-admin` tool. ::
```

Comment:
This reads like OSSEC will be using `securedrop-admin` to send the alert. Maybe:
```
OSSEC will send an alert when the `securedrop-admin` tool is used to backup, restore, or change the system configuration::
```

Reply:
fixed